src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --font-color: #3d4e61;
+  --white-color: #fff;
+  --border-color: #e1e7ed;
+}
+
+body {
+  margin-top: 0;
+  padding-top: 0;
+  font-family: "Avenir", sans-serif;
+  color: var(--font-color);
+  font-weight: 300;
+}
+
+.big-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.big-form__input-container,
+.small-form__input-container {
+  position: relative;
+  display: flex;
+}
+
+.big-form__icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.big-form__img {
+  width: 19px;
+  height: 19px;
+}
+
+.big-form__input,
+.small-form__input {
```

Comment:
you did an optimization, but read the checklist point 2 to get know how to do it better (instead of writing `.big-form__input` here and overwriting part of properties for it below)